util/Fs.cpp

```diff
@@ -184,7 +186,10 @@ ResourcePressure Fs::readRespressure(const std::string& path) {
 
 int64_t Fs::readMemcurrent(const std::string& path) {
   auto lines = readFileByLine(path + "/" + kMemCurrentFile);
-  OCHECK(lines.size() == 1);
+  if (!(lines.size() == 1))
+  {
+    throw "readMemcurrent (short lived process)";
```

Comment:
Throwing `char *` is usually not a good idea. Try creating a generic exception type in `include/Types.h` and throw that instead. That way, different errors can be handled or propagated up the stack.